packages/bot-web-ui/src/components/dashboard/tutorial-tab/faq-content.tsx

```diff
@@ -21,7 +23,8 @@ const FAQ = ({ type, content, src }: TDescription) => {
     return (
         <Text
             as='p'
-            line_height='xxl'
+            size={isMobile() ? 'xs' : 's'}
```

Comment:
@sandeep-deriv 
please use one call if it is possible instead of a few
` const is_mobile = isMobile();`

Reply:
done ✅